chat/vulkan/arm64/Containerfile

```diff
@@ -23,15 +23,15 @@ RUN dnf install -y python3-dnf-plugin-versionlock && \
     dnf install -y \
 https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
     dnf copr enable -y slp/mesa-krunkit epel-9-aarch64 && \
-    dnf install -y mesa-vulkan-drivers-23.3.3-102.el9.aarch64 && \
-    dnf versionlock  mesa-vulkan-drivers-23.3.3-102.el9.aarch64 && \
-    dnf install -y git cmake ninja-build gcc gcc-c++ vulkan-loader-devel vulkan-tools && \
+    dnf install -y mesa-vulkan-drivers-24.1.2-101.el9.aarch64 && \
+    dnf versionlock  mesa-vulkan-drivers-24.1.2-101.el9.aarch64 && \
+    dnf install -y git cmake ninja-build gcc gcc-c++ vulkan-loader-devel vulkan-tools fmt-devel && \
     dnf copr enable -y jeffmaury/shaderc epel-9-aarch64 && \
```

Comment:
This part was added recently because it was required by the llama-cpp Vulkan build, is it still required for kompute ?

Reply:
Yes, I updated the mesa package because el9 has switched to a newer llvm.

Reply:
Sorry I was not clear enough. I was mentioning the sharerc/glslc part

Reply:
Ah, yes, kompute also needs glslc